### PR TITLE
Status: 2023: OpenSSL 3, base: markup, changes

### PR DESCRIPTION
**Pull request by grahamperrin**
AsciiDoc: ' -- ' instead of ' - '.

Two updates are not a single update.

### CODE REVIEW

File: website/content/en/status/report-2023-07-2023-09/openssl3.adoc
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/openssl3.adoc:1]: Uppercase I for Improved? If there'll be Americanisation of headings.

Comment (lsalvadore) [reply website/content/en/status/report-2023-07-2023-09/openssl3.adoc:1]: I would go with uppercase I.

**Review by lsalvadore — APPROVED** [review]
Approved once the original author @khorben also approves.

**Comment by khorben** [comment]
> Approved once the original author @khorben also approves.

I'm happy either way.

**Pull request closed without merging**
